nautobot/apps/config.py

```diff
@@ -7,6 +7,6 @@
 def get_app_settings_or_config(app_name, variable_name):
     """Get a value from Django settings (if specified there) or Constance configuration (otherwise)."""
     # Explicitly set in settings.py or nautobot_config.py takes precedence, for now
-    if variable_name in settings.PLUGINS_CONFIG[app_name]:
-        return settings.PLUGINS_CONFIG[app_name][variable_name]
-    return getattr(config, f"{app_name}__{variable_name}")
+    if variable_name.lower() in settings.PLUGINS_CONFIG[app_name]:
+        return settings.PLUGINS_CONFIG[app_name][variable_name.lower()]
+    return getattr(config, f"{app_name}__{variable_name.upper()}")
```

Comment:
Should this be `f"{app_name}__{variable_name}".upper()` instead? Seems weird to me to force upper case for half of the name and expect lower case for the other half.

Reply:
I would defer to you, I thought this was a constance thing that we as users didn't have control over. When I tested creating it, this is the format it used.

* Constance says, "must be capital"
* I never set anything on the app_name, other than potentially what `nautobot_golden_config.confi.name` is??